.github/workflows/lint.yml

```diff
@@ -207,8 +207,8 @@ jobs:
         conda activate "${CONDA_ENV}"
 
         # Test tools
-        python3 -m unittest discover -vs tools/test -p 'test_*.py'
-        python3 -m unittest discover -vs .github/scripts -p 'test_*.py'
+        PYTHONPATH=$(pwd) pytest tools/test/test_*.py
+        PYTHONPATH=$(pwd) pytest .github/scripts/test_*.py
```

Comment:
Isn't this only setting they PYTHONPATH env var,  resulting in no test files being executed?

Reply:
Good catch!  The test works but one test there is not working with pytest https://github.com/pytorch/pytorch/actions/runs/12264094674/job/34217191327?pr=142476#step:14:135, so let me fix it